Decred Wallet/Features/Wallet Setup/ConfirmNewWalletSeedViewController.swift

```diff
@@ -90,8 +90,12 @@ extension ConfirmNewWalletSeedViewController: UITableViewDataSource{
         cell?.onPick = {(index, seedWord) in
             self.selectedSeedWords[indexPath.row] = index
             self.enteredWords[indexPath.row] = seedWord
-            if indexPath.row < 32{
-                tableView.selectRow(at: IndexPath(row: indexPath.row + 1, section: 0), animated: true, scrollPosition: .middle)
+            if indexPath.row < 32 {
+                if indexPath.row < 29 {
+                tableView.selectRow(at: IndexPath(row: indexPath.row + 2, section: 0), animated: true, scrollPosition: .bottom)
+                } else {
+                    tableView.selectRow(at: IndexPath(row: indexPath.row - 1, section: 0), animated: true, scrollPosition: .middle)
+                }
```

Comment:
Aside the formatting issues in this change (line 95 indenting), this if block can be simplified thus:
```
if indexPath.row < 29 {
    tableView.selectRow(at: IndexPath(row: indexPath.row + 2, section: 0), animated: true, scrollPosition: .bottom)
} else if indexPath.row < 32 {
    tableView.selectRow(at: IndexPath(row: indexPath.row - 1, section: 0), animated: true, scrollPosition: .middle)
}
```

Also, what's the significance of the **magic number** 29?